### PR TITLE
Implement complete method in iteration model

### PR DESCRIPTION
**Pull request by musayann**
Implemented a method which will tell if the iteration is complete in
case the iteration does not have a next question

### CODE REVIEW

File: app/models/iteration.rb [hunk 1]
```diff
@@ -13,19 +13,30 @@ def answer_question(answer, value = nil)
     iteration_answers.create(answer: answer, question: answer.question, value: value)
   end
 
+  
+  def complete?
```
Comment (musayann) [inline app/models/iteration.rb]: I'm not sure about the naming @bumi

Comment (bumi) [reply app/models/iteration.rb]: I'd called it `completed?` - I think most "states" are best described with past tense.

File: app/models/iteration.rb [hunk 2]
```diff
@@ -13,19 +13,27 @@ def answer_question(answer, value = nil)
     iteration_answers.create(answer: answer, question: answer.question, value: value)
   end
 
+  def complete?
+    return false if answers.last.blank? || answers.last.next_question_id
+
+    true
```
Comment (bumi) [inline app/models/iteration.rb]: this method could be written as: 

```ruby
def completed?
  answers.any? && answers.last.next_question_id.blank?
end
```

File: app/models/iteration.rb (hunk 1 above)
Comment (musayann) [reply app/models/iteration.rb]: Yeah, Good to know

**Comment by musayann** [comment]
@bumi  should we merge this one ??

**Pull request merged**